src/messages.coffee

```diff
@@ -0,0 +1,32 @@
+module.exports =
+  DEVICE_STATUS: (device) ->
+    unavailable = device.status == 'unavailable'
+    hasOwner = device.owner != 'taqtile'
+
+    output = []
+
+    output.push if unavailable then ':red_circle:' else ':large_blue_circle:'
+    output.push "id: `#{device.id}`"
+    output.push "name: `#{device.model}`"
+    output.push "version: `#{device.version}"
+    output.push "owned by _#{device.owner}_" if hasOwner
+    output.push "with #{device.user}" if unavailable
+
+    output.join ' '
+
+  SUCCESS_CREATE_DEVICE: (device) -> "Success adding `#{device.id}` (`#{device.model}`)!"
+
+  ERROR_GET_DEVICE: (id) -> "Device not found for id `#{id}`"
+
+  # got-devices
+  ERROR_DEVICE_OWNED: "It's already with you... o.O"
```

Comment:
You could keep the consistency here and have `device` as a param, specially in case you ever want to change the text message to have the device name